.travis.yml

```diff
@@ -37,9 +37,12 @@ script:
       make build || travis_terminate 1;
       make coverage || travis_terminate 1;
     fi
+  - make docker-build
 
 after_script:
-  - ./lostromos version
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
```

Comment:
This should be a deploy. https://docs.travis-ci.com/user/deployment/script/

We should build on `branch:master` and `tags:true`

Reply:
It should also support building images for the tag when we are on a tag and pushing those with the specific version.